src/vs/editor/contrib/goToDeclaration/goToDeclaration.ts

```diff
@@ -50,19 +66,19 @@ function getDefinitions<T>(
 }
 
 
-export function getDefinitionsAtPosition(model: IReadOnlyModel, position: Position): TPromise<Location[]> {
+export function getDefinitionsAtPosition(model: IReadOnlyModel, position: Position): TPromise<DefinitionAndSpan[]> {
```

Comment:
Change currently breaks the `vscode.executeDefinitionProvider` command. It previously returned `Location[]` but will now return a `DefinitionAndSpan[]`